libs/zjolt/libs/JoltC/JoltC.h

```diff
@@ -151,13 +167,76 @@ typedef struct JPH_BroadPhaseLayerInterfaceVTable JPH_BroadPhaseLayerInterfaceVT
 typedef struct JPH_BodyActivationListenerVTable   JPH_BodyActivationListenerVTable;
 typedef struct JPH_ContactListenerVTable          JPH_ContactListenerVTable;
 
+
+//--------------------------------------------------------------------------------------------------
+//
+// Structures
+//
+//--------------------------------------------------------------------------------------------------
+// NOTE: Needs to be kept in sync with JPH::AABox
+struct JPH_AABox
+{
+    alignas(16) float  min[3];
+    alignas(16) float  max[3];
+};
+
+// NOTE: Needs to be kept in sync with JPH::Plane
+struct JPH_Plane
+{
+    alignas(16) float  normal_and_constant[3];
+};
+
+// NOTE: Needs to be kept in sync with JPH::Sphere
+struct JPH_Sphere
+{
+    float  center[3];
+    float  radius;
+};
+
 // NOTE: Needs to be kept in sync with JPH::MassProperties
 struct JPH_MassProperties
 {
     float              mass;
     alignas(16) float  inertia[16];
 };
 
+// NOTE: Needs to be kept in sync with JPH::MotionProperties
+// TODO: Fix me! Incorrect size and/or incorrectly aligned members
+struct JPH_MotionProperties
+{
+    // 1st cache line
+    // 16 byte aligned
+    alignas(16) float   linear_velocity[3];
+    alignas(16) float   angular_velocity[3];
+    alignas(16) float   inv_inertia_diagnonal[3];
+    alignas(16) float   inertia_rotation[4];
+
+    // 2nd cache line
+    // 4 byte aligned
+    float    force[3];
+    float               torque[3];
+    float               inv_mass;
+    float               linear_damping;
+    float               angular_daming;
+    float               max_linear_velocity;
+    float               gravity_factor;
+    uint32_t            index_in_active_bodies;
+    uint32_t            island_index;
+
+    // 1 byte aligned
+    JPH_MotionQuality   motion_quality;
+    bool                allow_sleeping;
+
+    // 3rd cache line (least freqently used)
+    // 4 byte aligned
+    JPH_Sphere          sleep_test_spheres[3];
+    float               sleep_test_timer;
+
+#ifdef JPH_ENABLE_ASSERTS
+    JPH_MotionType      cached_motion_type;
+#endif
+};
```

Comment:
I'm having some trouble to get the size of the struct to be correct. Any ideas @michal-z?

Reply:
> I'm having some trouble to get the size of the struct to be correct. Any ideas @michal-z?

I think this is the last thing and then this PR is ready